### ISSUE TITLE
Thanks a lot for this!

### ISSUE THREAD

**Issue by sthewissen** [Zarbuz/VoxMerger#4]
First off, thanks a lot for this project! I've had to do some automation of merging vox files into a lot of different combinations, and this piece of software has really helped make that a reality. So massive kudos there!

One final question I had hoped maybe you had an answer too or know more about:

What would be the possibilities to perform a union at the end of the voxel merge to make the entire piece into a single piece instead of single items?

**Comment by Zarbuz** [comment]
I wrote VoxMerger before a release of MagicaVoxel where the importing of .vox bigger than 256 was impossible. The interest of this program is therefore a little less since the last updates of MagicaVoxel.
Usually I work on different parts of a model in different projects, and when the time comes to do some final rendering, you have to be able to import all the parts at once.

**Comment by sthewissen** [comment]
Yeah, I had to automate creating 5000 uniquely different models from a folder full of randomized .vox files, so combining with randomized input works great through this. I just don't feel like having to open 5000 individual files to Union them in MagicaVoxel, so I'd be happy to take any pointers on if/how that could be achieved.

**Comment by Zarbuz** [comment]
Well 5000 models in only one model could be too much, you will probably crash MagicaVoxel and/or VoxMerger. 

Put few models into a folder and try combine it by enter this command : ./VoxMerger.exe --i YourFolder --o Result

YourFolder should be the path of your folder

**Comment by sthewissen** [comment]
It's not that I'm making a model consisting of 5000 parts. I'm making 5000 unique models built from 10-20 random parts out of a big library. That part is already covered nicely by your tool! :) 

If I put 10 files in the input folder, it makes a nice single VOX file. However, those 10 random parts are separate parts in the resulting VOX file that can be moved individually, but I'd like them to be a single model. The same as selecting them all in MagicaVoxel and pressing U to union them. But I'd like to do that last step programmatically :)

**Comment by Zarbuz** [comment]
Ah I see ! VoxMerger is not able to do that at the moment sorry. Will try to implement an option to union or not all the models
